src/encoded/renderers.py

```diff
@@ -331,6 +400,22 @@ def should_transform(request, response):
     if response.content_type != 'application/json':
         return False
 
+    # cgap uses the following extra check that I think is the right thing, but leaving this commented out
```

Comment:
This is the other significant difference between `cgap-portal` and `fourfront`. See the comment text for details. This makes `test_should_transform` in `test_renderers.py` pass properly.